.github/workflows/validation.yml

```diff
@@ -0,0 +1,94 @@
+name: Gen3utils Validation
+
+on:
+  pull_request:
```

Comment:
in travis there were issues, but maybe with gh actions we could get it to run on `push` events too